### PR TITLE
Creation of Team SUSE


### PR DESCRIPTION
**Pull request by evrardjp**
#### ISSUE TYPE
feature pull request

#### SUMMARY
I hope, in the future, that a team of SUSE employees and openSUSE
community members will be stepping up to help maintain not
only zypper modules, but other modules impacting (open)SUSE.

For this, it makes sense to create a team.

I will be the first person of this organisation, and I hope I
will be joined by my colleagues soon.

### CODE REVIEW

**Review by mkrizek — APPROVED** [review]

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -206,6 +206,7 @@ files:
     maintainers: chouseknecht maxamillion fabianvf flaper87 willthames
     labels: k8s
   $modules/clustering/k8s/_kubernetes.py: supertom
+  $modules/clustering/etcd3.py: $team_suse
```
Comment (samdoran) [inline .github/BOTMETA.yml]: This probably should not be `$team_suse`. It's fine if you would like to individually added as a maintainer for `etcd3`, just add your name.

**Review by samdoran — CHANGES_REQUESTED** [review]

File: .github/BOTMETA.yml (hunk 1 above)
Comment (jctanner) [reply .github/BOTMETA.yml]: @evrardjp please follow sam's advice and change this entry.

Comment (evrardjp) [reply .github/BOTMETA.yml]: Will do.

**Comment by evrardjp** [comment]
@samdoran @jctanner  This is now fixed. Can you review again?

Sorry for the (relatively) long process.

**Comment by evrardjp** [comment]
We can probably remove the incorrect labels too.

**Review by samdoran — APPROVED** [review]

**Pull request merged**